### PR TITLE
US map

### PR DESCRIPTION
**Pull request by cluhring**

### CODE REVIEW

**Comment by cluhring** [comment]
Lydia's Search Trails Button isn't working :disappointed:

**Comment by Lydias303** [comment]
Yeah im not quite sure why it isnt working here. It def doesnt work on the trail show and really shouldn't be there at all but that was my mistake putting it in the application.hbs. It has been removed :). Lets get it fixed tomorrow before we merge.

File: app/components/us-map.js [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+  map: null,
+  classNames: ['us-map'],
+
+  didInsertElement: function () {
+
+    L.mapbox.accessToken = 'pk.eyJ1IjoiY2x1aHJpbmciLCJhIjoiNWF2Z1l6ZyJ9.8peAq7kTQyvXShlVv1K82w';
+
+    let map = L.mapbox.map(this.elementId, 'cluhring.9d2c52ea');
+    this.set('map', map);
+
```
Comment (Lydias303) [inline app/components/us-map.js:19]: Can you pull this data out into a seperate file? like statedata.json?

File: app/templates/trails.hbs [hunk 2]
```diff
@@ -1,3 +1,16 @@
 
 
+<div id="us-map" class="map">
+  <h3>Focused State: {{selectedState}}</h3>
+  {{us-map trails=model selectedState=selectedState}}
+</div>
+
```
Comment (Lydias303) [inline app/templates/trails.hbs]: Do we want a HI THERE?

File: tests/unit/components/us-map-test.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  assert.expect(2);
+
+  // Creates the component instance
+  var component = this.subject();
+  assert.equal(component._state, 'preRender');
+
+  // Renders the component to the page
+  this.render();
+  assert.equal(component._state, 'inDOM');
+});
```
Comment (Lydias303) [inline tests/unit/components/us-map-test.js:21]: nice testing :+1:

**Pull request merged**